Updating Loose Files and PSTs folder structure (#7905)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/export-documents-from-review-set.md

  - Warnings and errors x of z.csv: This file includes information about errors encountered when trying to export from the review set.
  
  - Exchange: This folder contains all content from Exchange stored in PST files. Redacted PDF files cannot be included with this option. If an attachment is selected in the review set, the parent email will be exported with the attachment attached.
      - This folder could also contain: 

>         - Any messages that were IRM protected and have been decoded. 

Reviewer: ```suggestion
        - Any Information Rights Management (IRM) protected messages that have been decoded. 
```

>         - Any messages that were error-remediated. 
>         - Any modern attachments/links referenced in e-mails. 

Reviewer: ```suggestion
        - Any error-remediated messages. 
        - Any modern attachments or links referenced in emails. 
```
